### PR TITLE
Fix error reported by flake8 5.0.1

### PR DESCRIPTION
**Pull request by JanuszL**
- fixes a few too long line
- fixes a lot of lack of space after a keyword (mostly assert)

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- fixes a few too long line
- fixes a lot of lack of space after a keyword (mostly assert)

### Affected modules and functionalities:
- lot of python files inside dali/

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Review by awolant — APPROVED** [review]

File: dali/test/python/sequences_test_utils.py [hunk 1]
```diff
@@ -224,8 +224,8 @@ def expand_arg(expandable_layout, arg_has_frames, input_batch, arg_batch):
             expanded_batch.extend(arg_sample for _ in range(num_frames))
         else:
             frame_idx = expandable_layout.find("F")
-            assert(frame_idx >= 0)
-            assert(len(arg_sample) == input_sample.shape[frame_idx])
+            assert (frame_idx >= 0)
+            assert (len(arg_sample) == input_sample.shape[frame_idx])
```
Comment (klecki) [inline dali/test/python/sequences_test_utils.py:228]: Drop the parenthesis everywhere?

Comment (JanuszL) [reply dali/test/python/sequences_test_utils.py:228]: Sure

Comment (JanuszL) [reply dali/test/python/sequences_test_utils.py]: Done for the problematic files.

**Comment by JanuszL** [comment]
!build

File: docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py [hunk 2]
```diff
@@ -325,11 +325,9 @@ def _get_caption_annotation(image_id):
 
 def main(_):
     assert FLAGS.image_dir, "`image_dir` missing."
-    assert (
-        FLAGS.image_info_file
-        or FLAGS.object_annotations_file
-        or FLAGS.caption_annotations_file
-    ), ("All annotation files are " "missing.")
+    assert FLAGS.image_info_file or \
```
Comment (klecki) [inline docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py:328]: I think here, the parenthesis might be preferred over `\`, but up to you.

File: docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py [hunk 3]
```diff
@@ -325,11 +325,9 @@ def _get_caption_annotation(image_id):
...
+           FLAGS.object_annotations_file or \
+           FLAGS.caption_annotations_file, ("All annotation files are " "missing.")
```
Comment (klecki) [inline docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py:330]: Small nitpick, looks like it is some leftover.
```suggestion
           FLAGS.caption_annotations_file, "All annotation files are missing."
```

**Review by klecki — APPROVED** [review]
Slight issue with one assert

File: docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py (hunk 3 above)
Comment (JanuszL) [reply docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py:330]: Will do in followup.

**Pull request merged**